File: Array/1528-kids-with-the-greatest-number-of-candies/kids-with-the-greatest-number-of-candies.java
class Solution {
    public List<Boolean> kidsWithCandies(int[] candies, int extraCandies) {
        

        int maxValue = Arrays.stream(candies).max().orElse(0);

        List<Boolean> out = new ArrayList<>();
        for(int num: candies) {
            if (num+extraCandies >= maxValue) {
                out.add(true);
            } else {
                out.add(false);
            }
        }

        return out;

    }
}

File: Array/1528-kids-with-the-greatest-number-of-candies/kids-with-the-greatest-number-of-candies.py
class Solution:
    def kidsWithCandies(self, candies: List[int], extraCandies: int) -> List[bool]:

        maxVal = max([x for x in candies])
        out = [True if x+extraCandies >= maxVal else False for x in candies  ]
        return out
       

        